main.lua

```diff
@@ -89,6 +89,8 @@ local function start(...)
     .alias({['A'] = 'auto-create-entity'})
     .describe("N", "do not prompt for entity creation")
     .alias({['N'] = 'no-entity'})
+    .describe("ca", "path for a custom CA certificate file in PEM format")
```

Comment:
Looks like you need to allocate a single letter here, like "C" doesn't seem to be taken (assuming it is case sensitive)